android/app/proguard-rules.pro

```diff
@@ -12,4 +12,5 @@
 -keep class com.facebook.jni.** { *; }
 -keep class com.swmansion.reanimated.** { *; }
 -keep class com.facebook.react.turbomodule.** { *; }
-
+# For more details, see the requirements setting for Kettle at https://developer.kogenta.com/docs/react-native/add-kettle#android
+-keepclassmembers class com.kogenta.kettle.core.model.** { *; }
```

Comment:
Will this actually have any effect? We don't have enabled proguard in production build now, as far as I know.

Reply:
I wasn't sure if is in use or not, but there are also another stuff set there, or Mapbox and Facebook, so we might be using it.

Reply:
At least we have `def enableProguardInReleaseBuilds = false` in build.gradle now. If Kogenta has a requirementet that the app should be obfuscated with proguard, we would need to look into enabling it.

It was originally planned in this issue, but not done yet:
https://github.com/AtB-AS/kundevendt/issues/3744

Reply:
Removed for now then ;)

Reply:
Aight. My comment wasn't mainly on whether this should be included or not, just that we need to be clear in communications with Kogenta that our app is not obfuscated on Android, and whether they have that requirement or not.

Reply:
In their SDK, they have mentioned: if Proguard is used, then add those rules. Doesn't sound like it is a requirement.

Reply:
Ok. Maybe check it up to be sure.